about/index.markdown

```diff
@@ -374,29 +374,4 @@ title: About
       </div>
     </div>
   </div>
-
-  <!--------------------------- Citing MoveIt --------------------------->
-  <!--------------------------------------------------------------------->
-  <!--------------------------------------------------------------------->
-  <!--------------------------------------------------------------------->
-  <!--------------------------------------------------------------------->
-  <h1>Citing MoveIt</h1>
-  <div class="container-fluid">
```

Comment:
Can you keep the h1 and just have a link to the new page?

Reply:
Done.